README.md

```diff
@@ -4,6 +4,25 @@
 
 A library providing a tuple data structure.
 
+## Status - complete
+
+We consider this package to be feature complete. With Dart 3.0, users now have
+the ability to use [Records](https://dart.dev/language/records):
+
+> Records are an anonymous, immutable, aggregate type. Like other collection
+  types, they let you bundle multiple objects into a single object. 
+
+```dart
+({int a, bool b}) record;
```

Comment:
[nit] I would show an example of a record with positional fields instead of named fields.

Reply:
updated -